feat(changelog): instances-fixed-zenbleed-incident-response 2023-07-28 (#1984)
scaleway/docs-content: changelog/july2023/2023-07-28-instances-fixed-zenbleed-incident-response.mdx

---
title: Zenbleed incident response
status: fixed
author:

>     fullname: 'Join the #community channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #instances channel on Slack.'
```

>     url: 'https://scaleway-community.slack.com/archives/C7Z76CCUE'

Reviewer: ```suggestion
    url: 'https://slack.scaleway.com'
```

date: 2023-07-28
category: compute
product: instances
---

> Addressed Zenbleed vulnerability on AMD processors within our DEV1, GP1, and VC Instance offers. Patching process completed, and all affected instances have been updated. Some users may experience slight performance impacts. For more details, read our blog post.

Reviewer 2: ```suggestion
Addressed Zenbleed vulnerability on AMD processors within our DEV1, GP1, and VC Instance offers. Patching process completed, and all affected Instances have been updated. Some users may experience slight performance impacts. For more details, read our [blog post](https://www.scaleway.com/en/blog/zenbleed-vulnerability/).
```

> Addressed Zenbleed vulnerability on AMD processors within our DEV1, GP1, and VC Instance offers. Patching process completed, and all affected instances have been updated. Some users may experience slight performance impacts. For more details, read our blog post.

Reviewer: ```suggestion
We have successfully mitigated the Zenbleed vulnerability affecting AMD processors across our DEV1, GP1, and VC Instance offerings. The patching process has been completed, and all impacted instances have been updated. While the security issue has been addressed, it's possible that some users may notice minor performance impacts. For further information, please refer to our comprehensive [blog post](https://www.scaleway.com/en/blog/zenbleed-vulnerability/).
```

> Addressed Zenbleed vulnerability on AMD processors within our DEV1, GP1, and VC Instance offers. Patching process completed, and all affected Instances have been updated. Some users may experience slight performance impacts. For more details, read our [blog post](https://www.scaleway.com/en/blog/zenbleed-vulnerability/).

Reviewer: ```suggestion
We have successfully mitigated the Zenbleed vulnerability affecting AMD processors across our DEV1, GP1, and VC Instance offerings. The patching process has been completed, and all impacted Instances have been updated. While the security issue has been addressed, it's possible that some users may notice minor performance impacts. For further information, please refer to our comprehensive [blog post](https://www.scaleway.com/en/blog/zenbleed-vulnerability/).
```

> We have successfully mitigated the Zenbleed vulnerability affecting AMD processors across our DEV1, GP1, and VC Instance offerings. The patching process has been completed, and all impacted Instances have been updated. While the security issue has been addressed, it's possible that some users may notice minor performance impacts. For further information, please refer to our comprehensive [blog post](https://www.scaleway.com/en/blog/zenbleed-vulnerability/).

Reviewer: ```suggestion
We have successfully mitigated the Zenbleed vulnerability affecting AMD processors across our DEV1, GP1, and VC Instance offerings. The patching process has been completed, and all impacted Instances have been updated. While the security issue has been addressed, it is possible that some users may notice minor performance impacts. For further information, please refer to our comprehensive [blog post](https://www.scaleway.com/en/blog/zenbleed-vulnerability/).
```

> We have successfully mitigated the Zenbleed vulnerability affecting AMD processors across our DEV1, GP1, and VC Instance offerings. The patching process has been completed, and all impacted Instances have been updated. While the security issue has been addressed, it is possible that some users may notice minor performance impacts. For further information, please refer to our comprehensive [blog post](https://www.scaleway.com/en/blog/zenbleed-vulnerability/).

Reviewer: ```suggestion
We have successfully mitigated the Zenbleed vulnerability affecting AMD processors across our DEV1, GP1, and VC Instance offerings. The patching process has been completed, and all impacted Instances have been updated. While the security issue has been addressed, it is possible that some users may notice minor performance impacts. For further information, please refer to our [blog post](https://www.scaleway.com/en/blog/zenbleed-vulnerability/).
```
